libwasmvm/src/cache.rs

```diff
@@ -719,18 +719,6 @@ mod tests {
             "0.2",
             "size_pinned_memory_cache: {size_pinned_memory_cache}"
         );
-        // todo check
```

Comment:
why is it deleted?

Reply:
This codes is test code of wasmvm v1.0.0. When I bump up to wasmvm v1.1.1, this codes was conflicted, so I changed it to a comment. 
And at the final check, it was confirmed that it was removed the upstream repository of v1.1.1. So I delete it.